LeaderboardBackend/Controllers/CategoriesController.cs

```diff
@@ -60,4 +61,90 @@ public async Task<ActionResult<CategoryViewModel>> CreateCategory(
             notFound => NotFound(ProblemDetailsFactory.CreateProblemDetails(HttpContext, 404, "Leaderboard Not Found"))
         );
     }
+
+    [Authorize(Policy = UserTypes.ADMINISTRATOR)]
+    [HttpPatch("category/{id:long}")]
+    [SwaggerOperation(
+        "Updates a category with the specified new fields. This request is restricted to administrators. " +
+        "Note: `type` cannot be updated." +
+        "This operation is atomic; if an error occurs, the category will not be updated. " +
+        "All fields of the request body are optional but you must specify at least one.",
+        OperationId = "updateCategory"
+    )]
+    [SwaggerResponse(204)]
+    [SwaggerResponse(401)]
+    [SwaggerResponse(403)]
+    [SwaggerResponse(404, Type = typeof(ProblemDetails))]
+    [SwaggerResponse(
+        409,
+        "The specified slug is already in use by another category. Returns the conflicting category.",
+        typeof(CategoryViewModel)
+    )]
```

Comment:
Should we use a Conflict Details here?

Reply:
Oh yeah of course. Sorry it slipped my mind. ~(Peer reviewing really does work, after all)~